client/src/components/Container.jsx

```diff
@@ -112,10 +112,10 @@ class Container extends React.Component {
     return (
       <div style={containerStyles.container}>
         <div style={{display: 'flex'}}>
-          <h2 style={containerStyles.header}>
+          <div style={containerStyles.header}>
```

Comment:
I didnt notice that you had `h2` in here before, good catch

Reply:
it looks a lot better as a div